toast-tk-runtime/src/main/java/io/toast/tk/runtime/AbstractScenarioRunner.java

```diff
@@ -122,6 +133,15 @@ public ITestPage runTestPage(final ITestPage testPage) throws IOException {
 		}
 		beginTest();
 		ITestPage result = runner.run(testPage);
+		
+		List<ITestPage> testPages = new ArrayList<ITestPage>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)